main/ZgatewayBT.ino

```diff
@@ -311,7 +311,7 @@ void MiLampDiscovery(char* mac) {
       //component type,name,availability topic,device class,value template,payload on, payload off, unit of measurement
   };
 
-  createDiscoveryFromList(mac, MiLampsensor, MiLampparametersCount);
+  createDiscoveryFromList(mac, MiLampsensor, MiLampparametersCount, "", "Xiaomi", "");
```

Comment:
You can find all the model reference below:
https://github.com/1technophile/OpenMQTTGateway/blob/e17b5b9c9d97ac83ef51aefdf8c93de408d8046e/main/ZgatewayBT.ino#L836

Reply:
fixed